src/actors/paddle_boat/update.inc.c

```diff
@@ -0,0 +1,6 @@
+#include <ultra64.h>
+#include <actor_types.h>
+
```

Comment:
This needs a comment explaining that this is only for the paddle wheel.
`// Rotates paddle boat's wheel.`